templates/vue/cypress/integration/lightbox/activity.spec.js

```diff
@@ -35,23 +65,6 @@ describe("Activity", () => {
 
       cy.tick(3600 * 1000) // move forward by 1 hour
       cy.contains("1:00:05").should("be.visible")
-
-      cy.route("POST", "**/quiz*").as("submit")
-      cy.route("GET", "**/entries*").as("entries")
-
-      cy.contains(/done/i).click()
-      cy.lightbox().within(() => {
-        cy.get("audio").should("be.visible")
-        cy.contains(/submit/i).click()
-        cy.contains(/submitting/i).should("be.visible")
-
-        cy.wait("@submit")
-        cy.wait("@entries")
-
-        cy.contains("Thanks!").should("be.visible")
-        cy.contains(/done/i).click()
-      })
-      cy.lightbox().should("not.exist")
     })
```

Comment:
Should we also add some tests here to ensure user can edit activities they have already answered? And maybe another test to test switching between questions in the same activity?